stdlib/public/Concurrency/AsyncCompactMapSequence.swift

```diff
@@ -14,6 +14,37 @@ import Swift
 
 @available(SwiftStdlib 5.5, *)
 extension AsyncSequence {
+  /// Creates an asynchronous sequence that maps the given closure over the
+  /// asynchronous sequence’s elements, omitting results that don't return a
+  /// value.
+  ///
+  /// Use the `compactMap(_:)` function to transform every element received from
```

Comment:
```suggestion
  /// Use the `compactMap(_:)` method to transform every element received from
```

Reply:
Nice catch. The [API review](https://github.com/apple/swift-evolution/blob/main/proposals/0298-asyncsequence.md) calls them functions, but in discussing with Philippe, we settled on method. (Combine's "operator" didn't make sense here, and there was never a "modifier" type usage like with SwiftUI.)